src/content/terraform/docs-landing.json

```diff
@@ -1,110 +1,82 @@
-					"url": "/vault/docs/secrets/kv"
+					"description": "Terraform creates and manages resources on cloud platforms and other services through their application programming interfaces (APIs).",
+					"title": "What is Terraform?",
+					"url": "/terraform/intro"
 				},
 				{
-					"description": "Generate database credentials dynamically based on configured roles.",
-					"title": "Database Credentials",
-					"url": "/vault/docs/secrets/databases"
+					"description": "Learn about popular Terraform use cases and related documentation you can use to create Terraform configurations and workflows.",
+					"title": "Use Cases",
+					"url": "/terraform/intro/use-cases"
 				},
 				{
-					"description": "Deploy Vault into Kubernetes using the official HashiCorp Vault Helm chart.",
-					"title": "Kubernetes Secrets",
-					"url": "/vault/docs/platform/k8s"
+					"description": "Learn how Terraform compares to other tools and services.",
+					"title": "Terraform vs. Alternatives",
+					"url": "/terraform/intro/vs"
 				}
 			]
 		},
 		{
 			"type": "card-grid",
-			"title": "Encryption Services",
-			"description": "Securely handle data such as social security numbers, credit card numbers, and other types of compliance-regulated information.",
+			"title": "Manage Infrastructure",
 			"cards": [
 				{
-					"description": "Apply cryptographic functions on data in-transit. Sign and verify data, generate hashes and HMACs of data, and act as a source of random bytes.",
-					"title": "Transit Secrets Engine",
-					"url": "/vault/docs/secrets/transit"
+					"description": "Describe infrastructure on various providers with Terraform's configuration language.",
+					"title": "Configuration Language",
+					"url": "/terraform/language"
 				},
 				{
-					"description": "Handle secure data transformation and tokenization against a provided input value using NIST vetted cryptographic standards and other transformations such as masking.",
-					"title": "Transform Secrets Engine",
-					"url": "/vault/docs/secrets/transform"
+					"description": "Use the Terraform CLI to manage configuration, plugins, infrastructure, and state.",
+					"title": "Transform CLI",
+					"url": "/terraform/cli"
 				},
 				{
-					"description": "Exchange a sensitive value for an unrelated value called a token. The original sensitive value cannot be recovered from a token alone but is irreversible.",
-					"title": "Tokenization",
-					"url": "/vault/docs/secrets/transform/tokenization"
+					"description": "Use CDK for Terraform to write and deploy Terraform configurations in your preferred programming language.",
+					"title": "CDK for Terraform",
+					"url": "/terraform/cdktf"
 				}
 			]
 		},
 		{
 			"type": "card-grid",
-			"title": "Key Management",
-			"description": "Use a standardized workflow for distribution and lifecycle management of cryptographic keys in various KMS providers.",
+			"title": "Develop and Share",
```

Comment:
I think this one should be "Collaborate"? for some reason I'm not sure the preview is up to date with the file I'm seeing here 🤔

Reply:
Yes, I've been having some trouble with this PR. I'm so sorry - let me get this all fixed up and I'll ping you with a fresh preview so you can more easily check it